client/bank/lcd/rest.go

```diff
@@ -17,5 +17,5 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec)
 		QueryTokenStatsRequestHandlerFn(cdc, cliCtx)).Methods("GET")
 
 	r.HandleFunc("/bank/accounts/{address}/transfers", SendRequestHandlerFn(cdc, cliCtx)).Methods("POST")
```

Comment:
/bank/accounts/{address}/transfers --> /bank/accounts/{address}/send

Reply:
A send transaction will add a transfers resouce, so I think it's reasonable.